modules/data-table/main/index.coffee

```diff
@@ -15,6 +21,203 @@ hx.userFacingText({
 fullWidthColSpan = 999 # the colspan used to make a cell display as an entire row
 collapseBreakPoint = 480
 
+columnOptionLookup = (options, name, id) ->
+  if options.columns isnt undefined and options.columns[id] isnt undefined and options.columns[id][name] isnt undefined
+    options.columns[id][name]
+  else
+    options[name]
+
+splitArray = (array, index) ->
+  left = if index is 0
+    []
+  else
+    array[0..index - 1]
+  right = if index is array.length - 1
+    []
+  else
+    array[index+1..array.length]
```

Comment:
Also may want `array[index+1...array.length]` here, otherwise you'll be indexing out of range (which will work since this gets converted into a slice, but is probably not what you want)